static-analysis/autodispose-lint/src/main/kotlin/com/uber/autodispose/lint/AutoDisposeDetector.kt

```diff
@@ -46,6 +46,11 @@ class AutoDisposeDetector: Detector(), SourceCodeScanner {
         Category.CORRECTNESS,
         10,
         Severity.ERROR,
+        // We use the overloaded constructor that takes a varargs of `Scope` as the last param.
+        // This is to enable on-the-fly IDE checks. We are telling lint to run on both
+        // JAVA and TEST_SOURCES in the `scope` parameter but by providing the `analysisScopes`
+        // params, we're indicating that this check can run on either JAVA or TEST_SOURCES and
+        // doesn't require both of them together.
```

Comment:
Link the source on this in lint-dev? Also can we remove when we have min AGP 3.3?

Reply:
You mean 3.4? Tor mentioned that his changes [will land in 3.4](https://groups.google.com/d/msg/lint-dev/ULQMzW1ZlP0/1dG4Vj3-AQAJ)

Reply:
errr yes 3.4 :)

Reply:
9276702618beb53936da4cec376d238a8d54ded6